### PR TITLE
Bugfix and tests for rgbToHex, new utility rgbToString.

### PR DESCRIPTION
**Pull request by sqsinger1123**
This PR:
- Fixes https://github.com/mexitek/phpColors/issues/25 - a bug where rgbToHex has problems converting RGB value '0'.
- implements tests for `rgbToHex` in `colorConvert.phpt`.
- adds utility method `rgbToString`, to convert an RGB array to its CSS string representation.
- fixes whitespace, indentation, & commenting typos.

### CODE REVIEW

File: src/Mexitek/PHPColors/Color.php [hunk 1]
```diff
@@ -196,9 +196,33 @@ public static function rgbToHex( $rgb = array() ){
         $hex[1] = str_pad(dechex($rgb['G']), 2, '0', STR_PAD_LEFT);
         $hex[2] = str_pad(dechex($rgb['B']), 2, '0', STR_PAD_LEFT);
 
+        // Make sure that 2 digits are allocated to each color.
+        $hex[0] = (strlen($hex[0]) == 1)? '0'.$hex[0] : $hex[0];
+        $hex[1] = (strlen($hex[1]) == 1)? '0'.$hex[1] : $hex[1];
+        $hex[2] = (strlen($hex[2]) == 1)? '0'.$hex[2] : $hex[2];
```
Comment (sqsinger1123) [inline src/Mexitek/PHPColors/Color.php]: These lines are the core of this bugfix.

Other changes implement and support tests, or cleanup.

**Comment by mexitek** [comment]
Thanks for the contribution!

**Comment by strarsis** [comment]
@mexitek: Please also release to packagist!

**Comment by mexitek** [comment]
@strarsis I just added you as a collaborator to this repo. Feel free to help me out. I haven't used PHP in about 6 years now. Feel free to take the reins as much as you'd like.

**Comment by mexitek** [comment]
@strarsis I actually already had an account with packagist, I just forgot. I updated the webhook so changes should be flowing through now. 

Thanks for all the contributions!

**Comment by sqsinger1123** [comment]
No worries, thanks for accepting my PR and welcoming me as a contributor 😄

**Pull request merged**